frontend/packages/kubevirt-plugin/src/utils/validations/vm/disk.ts

```diff
@@ -129,8 +133,11 @@ export const validateDisk = (
       addRequired(pvcName);
       validations.pvc = validatePVCName(pvcName, usedPVCNames);
     }
-  }
 
+    // TODO: implement CDROM disk bus validation
```

Comment:
I meant, lets revert it to the same state it was before. Thumbs up for the TODO.

```suggestion
    if (disk.getType() === DiskType.DISK) {
      addRequired(allowedBusses.has(disk.getDiskBus()));
      validations.diskInterface = validateBus(disk.getDiskBus(), allowedBusses);
    }
```